variables.tf

```diff
@@ -201,3 +213,14 @@ variable "circuit_breaker_sns_topic_arn" {
   default     = null
   description = "The arn of the SNS topic to publish deployment circuit breaker failure messages to. If not provided, a SNS topic will be provided by this module."
 }
+
+variable "network_configurations" {
+  type        = list(object({ subnets = list(string), security_groups = list(string), assign_public_ip = bool }))
+  description = "A list of definitions to attach network configurations to the ECS task."
+}
```

Comment:
I think it would be easier to read the type here if there were some line breaks.

Reply:
Also, does this variable break things without a value?

Reply:
Yeah fair, I copied this over from our Fargate singleton module: https://github.com/7Factor/terraform-fargate-ecs-singleton-task/blob/master/variables.tf 
Let me see how it interacts if there's no value provided. I think if I give it a default of an empty list it should be fine.

Reply:
```suggestion
variable "network_configurations" {
  type = list(object({
    subnets          = list(string),
    security_groups  = list(string),
    assign_public_ip = bool
  }))
  description = "A list of definitions to attach network configurations to the ECS task."
}

```

Reply:
Updated to include a default value of an empty list and prettified. Might be worth going through after this PR in a follow-up to fix formatting, every pre-existing list is smushed onto one line.